full/src/test/java/org/apache/metamodel/ExcelDataContextFactoryTest.java

```diff
@@ -23,9 +23,7 @@
 import org.apache.metamodel.excel.ExcelConfiguration;
 import org.apache.metamodel.excel.ExcelDataContext;
 
-import junit.framework.TestCase;
-
-public class DataContextFactoryTest extends TestCase {
+public class ExcelDataContextFactoryTest extends  MongoDbTestCase{
```

Comment:
I know this class currently seems to only test the excel scenario, but it is a test of ```DataContextFactory``` so I find it more natural to name it like that?